recipes/nibabel/run_test.py

```diff
@@ -0,0 +1,14 @@
+import nose
+import sys
+import re
+
+splatform = sys.platform
+if splatform.startswith('win32'):
+    # Installation of data files for script testing a bit rickety on Appveyor:
+    config = nose.config.Config(verbosity=2,
+                                exclude=[re.compile("test_scripts"),
+                                         re.compile("test_dft")])
+else:
+    config = nose.config.Config(verbosity=2)
```

Comment:
Verbosity reduced! The nose output now seem to be going off the right side of the logs on Appveyor :-/

Failures remain only on Python 3.5/Windows. Any ideas?